tools/awk_gender_correction.cwl

```diff
@@ -0,0 +1,145 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: awk_gender_correction
+doc: |
+  TGEN gender correction 
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'staphb/bcftools:1.10.2'
+  - class: InitialWorkDirRequirement
+    listing:
+    - entryname: coyote_omega_awk.sh 
```

Comment:
Ha!

Reply:
That's what the cwl used to be called before I started cleaning up the color commentary.